src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/pooling.cpp

```diff
@@ -287,6 +495,24 @@ const std::vector<maxPoolV8SpecificParams>& paramsMaxV84D() {
     return paramsMaxV84D;
 }
 
+const std::vector<maxPoolV8SpecificParams>& paramsMaxV144D() {
+    static const std::vector<maxPoolV8SpecificParams> paramsMaxV144D = {
+            maxPoolV8SpecificParams{ {2, 2}, {2, 2}, {1, 1}, {0, 0}, {0, 0},
+                                                            ov::element::Type_t::i32, 0,
+                                                            ov::op::RoundingType::CEIL, ov::op::PadType::SAME_LOWER },
```

Comment:
Is there any test with RoundingType::FLOOR and MaxPool-14?

Reply:
Yes, for both AvgPool I'm testing the FLOOR/CEIL and CEIL_TORCH cases separately due to a fallback to reference plugin when CEIL_TORCH meets dynamic shapes.